src/main/java/ch/njol/skript/expressions/ExprExperienceCooldown.java

```diff
@@ -0,0 +1,98 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Experience Cooldown")
+@Description({
+	"The experience cooldown of a player.",
+	"Experience cooldown is how long until a player can pick up another orb of experience."
```

Comment:
Should clarify if this is constantly changing/counting down, or if it's the "default" cooldown that's started every time an orb is picked up. I'm not sure which it is tbh, but it seems like the first.